dev/io.openliberty.org.hibernate.validator.7.0/bnd.overrides

```diff
@@ -0,0 +1,87 @@
+Bundle-SymbolicName: io.openliberty.org.hibernate.validator.7.0
+Bundle-Description: Hibernate's Bean Validation (JSR-380) reference implementation; version=7.0.0.Alpha2
+
+# Provide these as resources to applications
+app-resources= \
+  META-INF/services/jakarta.validation.spi.ValidationProvider | \
+  META-INF/validation-mapping-1.0.xsd | \
+  META-INF/validation-configuration-1.0.xsd | \
+  META-INF/validation-mapping-1.1.xsd | \
+  META-INF/validation-configuration-1.1.xsd | \
+  META-INF/validation-mapping-2.0.xsd | \
+  META-INF/validation-configuration-2.0.xsd | \
+  META-INF/validation-mapping-3.0.xsd | \
+  META-INF/validation-configuration-3.0.xsd
+
+Export-Package: \
+  org.hibernate.validator.*;version="7.0.0.Alpha2";thread-context=true
+
+Import-Package: \
+  !jakarta.persistence, \
+  jakarta.validation;version="[3.0.0,4.0.0)", \
+  jakarta.validation.constraints;version="[3.0.0,4.0.0)", \
+  jakarta.validation.constraintvalidation;version="[3.0.0,4.0.0)", \
+  jakarta.validation.executable;version="[3.0.0,4.0.0)", \
+  jakarta.validation.groups;version="[3.0.0,4.0.0)", \
+  jakarta.validation.metadata;version="[3.0.0,4.0.0)", \
+  jakarta.validation.spi;version="[3.0.0,4.0.0)", \
+  jakarta.validation.valueextraction;version="[3.0.0,4.0.0)", \
+  javax.script;version=0, \
+  !javax.xml.bind.*, \
+  javax.xml.namespace;version=0, \
+  javax.xml.stream;version=0, \
+  javax.xml.stream.events;version=0, \
+  javax.xml.transform;version=0, \
+  javax.xml.transform.stream;version=0, \
+  javax.xml.validation;version=0, \
+  jakarta.el;version="[4.0.0,5.0.0)";resolution:=optional, \
+  org.xml.sax;version=0, \
+  org.jboss.logging;version="[3.1.0,4.0.0)", \
+  com.fasterxml.classmate;version="[1.3,2.0.0)", \
+  com.fasterxml.classmate.members;version="[1.3,2.0.0)", \
+  org.joda.time;version="[2.0.0,3.0.0)";resolution:=optional, \
+  org.jsoup;version="[1.5.2,2.0.0)";resolution:=optional, \
+  org.jsoup.nodes;version="[1.5.2,2.0.0)";resolution:=optional, \
+  org.jsoup.parser;version="[1.5.2,2.0.0)";resolution:=optional, \
+  org.jsoup.safety;version="[1.5.2,2.0.0)";resolution:=optional, \
+  javax.money;version="[1.0.0,2.0.0)";resolution:=optional, \
+  com.thoughtworks.paranamer;version="[2.5.5,3.0.0)";resolution:=optional, \
+  com.ibm.wsspi.classloading,\
+  org.apache.el
+
+# Use dynamicImport-Package for JAXB APIs, with this, Equonix will have a chance to wire
+# those packages to the JAXB-2.2 API if the target bundle is started, or the one from system
+# bundle will be always used due to JAXB 2.2 API is not resolved. Similarly, do the same
+# with persistence because the open source optionally uses jpa if it is there for a customized
+# TraversableRevolver implementation.
+DynamicImport-Package: \
+ jakarta.persistence, \
+ javax.xml.bind, \
+ javax.xml.bind.annotation, \
+ javax.xml.bind.annotation.adapters, \
+ javax.xml.bind.attachment, \
+ javax.xml.bind.helpers, \
+ javax.xml.bind.util
```

Comment:
Maybe add a TODO to replace all of the jaxb related imports/dynamic-imports with standard imports for jaxb 3.0 (which should be getting delivered in liberty soon, not sure if bval supports it yet though)

Reply:
Added the following:
`# TODO: Replace all of the jaxb related dynamic-imports with standard imports for JAXB 3.0 once available.`

Reply:
Correction.  JAXB is no longer needed by Hibernate Validator.  I've cleaned up the bnd.overrides and feature accordingly.